src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -6173,11 +6202,10 @@ void ObjectMgr::LoadAccessRequirements()
                     if (!sAchievementStore.LookupEntry(progression_requirement->id))
                     {
                         sLog->outErrorDb("Required achievement %u for faction %u does not exist for map %u difficulty %u, remove or fix this achievement requirement.", progression_requirement->id, requirement_faction, mapid, difficulty);
-                        delete progression_requirement;
```

Comment:
why?

Reply:
I changed it to make it easier to see when its being deleted and when not.